prx_simulation/control_msg[] plan
float64[] end_state
string system_name
